web-client/iron-remote-gui/.prettierrc

```
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
+	"singleQuote": true,
+	"trailingComma": "none",
```

Let’s use `all`, like in Rust
```suggestion
	"trailingComma": "all",
```

---

updated